Sources/WeaverCodeGen/RegexUtils.swift

```diff
@@ -32,9 +32,11 @@ enum Patterns {
     private static let equal = "\(spaces)=\(spaces)"
     private static let arrow = "\(spaces)<-\(spaces)"
     private static let name = "\\w+"
+    private static let typeNamePart = "\(name)(\\.\(name))*"
```

Comment:
I'm concerned that this won't allow arrays or dictionaries to reference nested types:
```
// weaver: foo <= [Bar.Baz]
// weaver: fuu <= [String: Bar.Baz]
```

Reply:
oh, thank you!
I'll take a look